### PR TITLE
Cleanup

### PR DESCRIPTION
**Pull request by alejandro-colomar**
A few more cleanup changes :)

### CODE REVIEW

File: lib/defines.h [hunk 1]
```diff
@@ -235,14 +205,6 @@ static inline void memzero(void *ptr, size_t size)
 # define SEEK_END 2
 #endif
 
-#ifdef STAT_MACROS_BROKEN
-# define S_ISDIR(x) ((x) & S_IFMT) == S_IFDIR)
-# define S_ISREG(x) ((x) & S_IFMT) == S_IFREG)
-# ifdef S_IFLNK
-#  define S_ISLNK(x) ((x) & S_IFMT) == S_IFLNK)
```
Comment (cgzones) [inline lib/defines.h:242]: Note that this macro doesn't even work due to unbalanced parenthesis.

Comment (alejandro-colomar) [reply lib/defines.h:242]: Good eye!  :)

**Comment by alejandro-colomar** [comment]
Oops, I moved my repos from github to my own cgit server <http://www.alejandro-colomar.es/src/>.  I forgot that I had this PR pending, and I don't know if I broke it, or if github keeps it in some cache for you.  If you can't merge the PR, tell me and I can recreate the repository, and push the branch.

Otherwise, if you accept patches to the mailing list, I'll be happy to send a patch set there (and in fact I'd prefer this, if that's okay for you).

**Comment by hallyn** [comment]
Thanks for doing all this research.

Github seems happy with this so I'll merge it.  I'm definitely happy taking patches over the mailing list, though.

I'm considering keeping a mirror of this tree at sr.ht, and creating a dev mailing list for it there, as that would make CI against patches easier.

**Comment by alejandro-colomar** [comment]
Great!  I'm happy to help :-)

**Pull request merged**